gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -565,6 +546,12 @@ public Builder setChannelPrimer(ChannelPrimer channelPrimer) {
       return this;
     }
 
+    /** Whether attempt DirectPath. */
+    public Builder setAttemptDirectPath(boolean attemptDirectPath) {
```

Comment:
This should have an annotation of either InternalApi or BetaApi. This should also have better docs explaining that this an experimental feature and is meant for client authors to enable when their service starts to support DirectPath